### PR TITLE
add text_generation keras example

### PR DESCRIPTION
**Pull request by Bob-Chou**
an error when run model.fit:
: java.lang.IllegalArgumentException: Node 'gradients/lstm/while/Exit_2_grad/b_exit' expects to be colocated with unknown node 'lstm/while/Exit_2'

### CODE REVIEW

File: pyzoo/zoo/examples/tensorflow/tfpark/text_generation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1269 @@
...
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "UK-hmKjYVoll"
+   },
+   "outputs": [],
+   "source": [
+    "keras_model = KerasModel(model)\n",
+    "history = keras_model.fit(input_example, target_example, \n",
```
Comment (hkvision) [inline pyzoo/zoo/examples/tensorflow/tfpark/text_generation.ipynb]: In our implementation currently fit doesn't return anything, so remove `history = `
